cmd/agent/commands/http.go

```diff
@@ -56,6 +66,7 @@ func BuildHTTPCmd() *cobra.Command {
 	cmd.Flags().StringVarP(&disruption.ErrorBody, "body", "b", "", "body for injected faults")
 	cmd.Flags().StringSliceVarP(&disruption.Excluded, "exclude", "x", []string{}, "comma-separated list of path(s)"+
 		" to be excluded from disruption")
+	cmd.Flags().BoolVar(&transparent, "transparent", true, "run as transparent proxy")
```

Comment:
The `transparent` flag seems good from an implementation point of view, but perhaps for users it could be a bit opaque (specially if they're not familiar with proxies). WDYT about naming the flag `disrupt` instead?

Reply:
I think `disrupt` is too ambiguous as it always disrupts. The different is how the traffic is intercepted. Also, consider that this option is intended for the developers of the disruptor, not the users of the disruptors. I expect someone trying to debug the disruptor to be familiar with the implementation details. 

All that considered, I would prefer to make the purpose of the option more explicit and leave it as is.

Reply:
> I think disrupt is too ambiguous as it always disrupts

🤔 Then I probably I didn't get this right. I thought that returning before creating and using the disruptor object meant no disruption, I'll need to give the code a deeper look to better understand what `proxy` does exactly. If disruption is hapenning unconditionally I'm happy with `transparent`.